distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Data/Vector.enso

```diff
@@ -231,10 +234,7 @@ type Vector
              [1, 2, 3, 4, 5].exists (> 3)
     exists : (Any -> Boolean) -> Boolean
     exists predicate =
-        len = this.length
-        go idx found = if found || (idx >= len) then found else
-            @Tail_Call go idx+1 (predicate (this.unsafe_at idx))
-        go 0 False
+        0.up_to this.length . exists (idx -> (predicate (this.unsafe_at idx)))
```

Comment:
```suggestion
        0.up_to this.length . exists idx->
            predicate (this.unsafe_at idx)
```

While the original is fine, I think that this syntax also works and IMO it is a bit cleaner. But that is just a subjective style thing.